README.md

```diff
@@ -542,6 +542,12 @@ to all Giant Swarm installations.
     - [v27.0.1](https://github.com/giantswarm/releases/tree/master/vsphere/v27.0.1)
     - [v27.0.0](https://github.com/giantswarm/releases/tree/master/vsphere/v27.0.0)
 
+## VMware Cloud Director
```

Comment:
I mean, you're totally right, as everywhere I searched for "CAPVCD" it states "Cluster API for VMware Cloud Director" and for "CAPV" they only call it "Cluster API for vSphere" without "VMware", but it still makes me a little angry. 🤣

Reply:
Don't get me wrong: No need to change, it's just my OCD.